validator/keymanager/imported/keymanager.go

```diff
@@ -100,6 +103,7 @@ func NewKeymanager(ctx context.Context, cfg *SetupConfig) (*Keymanager, error) {
 	if err := k.initializeAccountKeystore(ctx); err != nil {
 		return nil, errors.Wrap(err, "failed to initialize account store")
 	}
+	// Is this deprecated code that can be removed?
```

Comment:
Let's not remove it in this PR. This is to support users that created a wallet before Prysm's beta, which are likely thousands of validators. We will remove this for mainnet v1 release

Reply:
I'll open an issue for this here https://github.com/prysmaticlabs/prysm/issues/7752